pkg/apis/core/validation/validation.go

```diff
@@ -1920,7 +1913,7 @@ func ValidatePersistentVolumeClaimUpdate(newPvc, oldPvc *core.PersistentVolumeCl
 		}
 	}
 
-	if utilfeature.DefaultFeatureGate.Enabled(features.BlockVolume) {
+	if oldPvc.Spec.VolumeMode != nil {
```

Comment:
revert until PVC prepareforcreate/prepareforupdate logic is updated

Reply:
add pvc handling to this pr